src/components/scale-button.js

```diff
@@ -73,6 +74,11 @@ AFRAME.registerComponent("scale-button", {
       if (this.isScaling || !this.objectToScale) {
         return;
       }
+
+      if (!(NAF.utils.isMine(this.networkedEl) || NAF.utils.takeOwnership(this.networkedEl))) {
```

Comment:
might need to check for this.networkedEl in case the getNetworkedEntity promise hasn't resolved yet

Reply:
The check above for `this.objectToScale` will early out in this case, since they're both given values in the `getNetworkedEntity` callback https://github.com/mozilla/hubs/pull/2442/files#diff-425a09cd704e35df9abb0e6b291f4685R74
I could add another check for `networkedEl` though if that reads clearer.